src/sidecar/connectionStatusUtils.ts

```diff
@@ -40,8 +39,18 @@ function isDirectConnectionStable(connection: Connection): boolean {
   const kafkaState = status.kafka_cluster?.state;
   const schemaRegistryState = status.schema_registry?.state;
 
-  const rv = kafkaState !== "ATTEMPTING" && schemaRegistryState !== "ATTEMPTING";
+  const kafkaFailedReason = status.kafka_cluster?.errors?.sign_in?.message;
+  const schemaRegistryFailedReason = status.schema_registry?.errors?.sign_in?.message;
+  if (kafkaFailedReason) {
+    logger.error(`isDirectConnectionStable(): kafka auth failed with error: ${kafkaFailedReason}`);
+  }
+  if (schemaRegistryFailedReason) {
+    logger.error(
+      `isDirectConnectionStable(): schema registry auth failed with error: ${schemaRegistryFailedReason}`,
+    );
```

Comment:
Minor: I think we can remove the `auth failed with` altogether since these aren't auth-related (seems odd to have it mentioned for the CCloud one as well, but that at least requires API key/secret)

Reply:
K, done, and made into a loop. And connectionLoading.fire(id) restored, and its observer guarded against being called for ccloud ids.